### ISSUE TITLE
Add credit for Justice image for redesigned homepage

### ISSUE THREAD

**Issue by JessicaLucindaCheng** [hackforla/website#3415]
### Overview
We need to add a credit for a new image used in the redesigned version of the homepage so that the Credits webpage is up to date.

### Details
From the Homepage Redesign Credits in Figma, here is a screenshot of the credit's information for reference:

  <details>
    <summary>Click here to see the screenshot of the Justice credit in Figma</summary>
    <img src="https://user-images.githubusercontent.com/31293603/180343018-2e686725-a1d2-4bf4-a5e6-be78b13ef3cc.png" alt="">
  </details>

### Action Items
- [x] **Feature Branch:** You will be working off a feature branch called `feature-homepage-launch`. Please follow ["Step 1: Create your branch" in How to work off of a feature branch](https://github.com/hackforla/website/wiki/How-to-work-off-of-a-feature-branch#step-1-create-your-branch) wiki article.
- [x] In `_data/internal/credits` directory, create a file called `justice.yml`
- [x] In the file you just created, copy and paste the following:
```
---
# The title from the source provider
title: Justice
# The url where you can get direct access to the image on the provider site
title-link: https://unsplash.com/photos/bvHcG6334dE
# The type of content - image, audio, or video 
content-type: image
# The page it's used on
used-in: Homepage
# Name of the artist
artist: Sean Lee
# Name of the image database you found it on
provider: Unsplash
# Please give the general site URL, not the link to the image
provider-link: https://unsplash.com/
# Location in our GitHub repo
image-url: /assets/images/program-areas/justice.png
# Alt text for the image on the Credits page
alt: ''
---
```
- **Note:** When you use Docker to view the Justice credit on your computer and click on the `Justice` link in the Name field, that link will lead to `Page not found`. We spoke to Bonnie about this and she said it was okay for that link for Justice to be that way.
- [x] To update your topic branch, please follow ["Step 2: Update your branch with upstream changes" in How to work off of a feature branch](https://github.com/hackforla/website/wiki/How-to-work-off-of-a-feature-branch#step-2-update-your-branch-with-upstream-changes) wiki article.
- [x] The pull request created from this issue should be merged into `feature-homepage-launch`. Please follow ["Step 3: Create a pull request" in How to work off of a feature branch](https://github.com/hackforla/website/wiki/How-to-work-off-of-a-feature-branch#step-3-create-a-pull-request) wiki article
- [x] Once the pull request associated with this issue is approved and merged, please update and edit epic #3130 by
  - [x] Checking off the dependency for this issue
  - [x] If all dependencies are checked off, please do the following in issue #3130:
    - [x] Move it to the New Issue Approval column
    - [x] Remove the "Dependency" label from it
    - [x] Add a "ready for dev lead" label to it  
  
### Resources/Instructions
- [Credits webpage](https://www.hackforla.org/credits/)
- [Homepage Redesign Credits in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=17951%3A122724)
- [How to work off of a feature branch wiki page](https://github.com/hackforla/website/wiki/How-to-work-off-of-a-feature-branch)
- [Credits yml file template for Iconography and Imagery](https://github.com/hackforla/website/wiki/Credits-yml-file-template-for-Iconography-and-Imagery)
- This issue is tracked in the epic #3130.

**Comment by JessicaLucindaCheng** [comment]
This issue would usually have the milestone 01. Compliance. However, it's specifically for the homepage launch, which has not happened yet.

**Comment by JessicaLucindaCheng** [comment]
The link to the specific image says, "Page not found". Here is the link: https://unsplash.com/photos/bvHcG6334dE

Is this okay?

**Comment by JessicaLucindaCheng** [comment]
> The link to the specific image says, "Page not found". Here is the link: https://unsplash.com/photos/bvHcG6334dE
> 
> Is this okay?

- Ans from Bonnie: Instead of a link to the image, find a link to the author's page and use that link instead of the link to the image

**Comment by JessicaLucindaCheng** [comment]
@ExperimentsInHonesty 
> > The link to the specific image says, "Page not found". Here is the link: https://unsplash.com/photos/bvHcG6334dE
> > Is this okay?
> 
> * Ans from Bonnie: Instead of a link to the image, find a link to the author's page and use that link instead of the link to the image

I tried looking for "Sean Lee" on Unsplash and the [results](https://unsplash.com/s/users/Sean-Lee) of the search shows multiple Sean Lees. I'm not sure which Sean Lee it is because I did not find the Justice image we used.
<details>
  <summary>Click here to see the screenshot of the Justice credit with the image in question</summary>
  <img src="https://user-images.githubusercontent.com/31293603/180343018-2e686725-a1d2-4bf4-a5e6-be78b13ef3cc.png" alt="">
</details>

What do you want me to do about the link we use for the `title-link` field?
Ans: From Bonnie, just use the link that was provided to the image. It is okay if it leads to a page not found.

**Comment by maryjng** [comment]
Availability: 1/17 9 PM - 10 PM EST, 1/18 5 PM EST
ETA: 1/17
